src/device.js

```diff
@@ -13,6 +15,14 @@ export function isDevice(userAgent? : string = getUserAgent()) : boolean {
     return false;
 }
 
+export function isTablet(userAgent? : string = getUserAgent()) : boolean {
```

Comment:
Change looks good to me. Could you add some unit tests for this as well? Similar to: https://github.com/krakenjs/belter/blob/master/test/tests/device/isDevice.js

Reply:
@gregjopa It looks like the tests were split into separate files.  I have included test in `test/tests/device/isTablet.js`

Reply:
https://github.com/krakenjs/belter/pull/63/files#diff-460a3c38bb5195e525ae7447c5e2f5a20aca7705defc2c6f4342a7860bef55d6

Reply:
Thanks for adding tests @TNuzzi! 💯